mmv1/products/workstations/WorkstationConfig.yaml

```diff
@@ -650,6 +650,12 @@ properties:
     description: |
       Disables support for plain TCP connections in the workstation. By default the service supports TCP connections via a websocket relay. Setting this option to true disables that relay, which prevents the usage of services that require plain tcp connections, such as ssh. When enabled, all communication must occur over https or wss.
     min_version: 'beta'
+  - name: 'maxUsableWorkstations'
+    type: Integer
+    description: |
+      Maximum number of workstations under this configuration a user can have workstations.workstation.use permission on. Only enforced on CreateWorkstation API calls on the user issuing the API request.
+    min_version: 'beta'
```

Comment:
Fly-by review as I just triaged the associated issue - this field appears to be available in the GA version of the API: https://cloud.google.com/workstations/docs/reference/rest/v1/projects.locations.workstationClusters.workstationConfigs#WorkstationConfig.FIELDS.max_usable_workstations

Reply:
Shd I remove the `min_version` field then?

Reply:
yes. We are adding support to GA field.

Reply:
done.